server/api/test/index.js

```diff
@@ -117,4 +117,24 @@ describe( 'api', function() {
 			.send( { username: 'validuser', password: 'validpassword' } )
 			.expect( 408, response, done );
 	} );
+
+	maybeIt( 'should return a 400 push verification request if the server supports it', function( done ) {
```

Comment:
Crazy - a better name would _definitely_ help here, as `maybeIt()` sounds like a useless test to run.

Then again, it isn't really related to desktop either, is it? I guess that's a valid name for now, but it seems far more related to oAuth than desktop…